inc/client.hpp

```diff
@@ -30,13 +25,9 @@ class Data_Retrieve
 		Data_Retrieve(int port);
 		Data_Retrieve();
 		void Connection_verify();
-		void Full_Data();
-		int bits();
+		int Poll_Server();
```

Comment:
These are exactly the things you need to consider @RaoAbdulSami  for the serialiser

Reply:
oky sir